processor/servicegraphprocessor/processor.go

```diff
@@ -102,9 +108,28 @@ func newProcessor(logger *zap.Logger, config component.Config) *serviceGraphProc
 		pConfig.VirtualNodePeerAttributes = defaultPeerAttributes
 	}
 
+	scopeName := "processor/servicegraphprocessor"
+	meter := set.MeterProvider.Meter(scopeName)
+
+	droppedSpan, _ := meter.Int64Counter(
```

Comment:
Idea: Since I see we started to use "metadata" yaml at leas for the type. Can we have in there a "component metrics" section (similar but different than the "scraper metrics" we already have) than essentially generate a function where we can pass a "MeterProvider" and creates all instruments and simplifies this boilerplate?

Kind of an "obsreport" per component custom generated via the metadata yaml?

Reply:
@bogdandrutu yeah that would be useful.... gets rid of the scope naming question as well

Reply:
@dmitryax since you are the most knowledgeable person about metadata gen.

Reply:
I love this idea. Otherwise we have more docs on those metrics to manage elsewhere.